extensions/rich_text_components/Video/Video.html

```diff
@@ -1,7 +1,8 @@
 <script type="text/ng-template" id="richTextComponent/Video">
   <center>
-    <iframe width="500" height="280"
-            ng-src="<[videoUrl]>" frameborder="0" allowfullscreen tabindex="<[tabIndexVal]>">
-    </iframe>
+    <div style="position: relative; padding-bottom: 60.9%;">
+      <iframe width="460" height="280" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"
```

Comment:
Can we pull the styles here and above out into CSS classes? I recommend having a `<style>` tag block at the bottom of the file, outside the script tags (so that it only gets replicated once). Probably prefix everything with e.g. `.oppia-noninteractive-video`, e.g. `.oppia-noninteractive-video-iframe` and `.oppia-noninteractive-video-iframe-container`.